pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -81,7 +84,55 @@ var (
 	DefaultCleanPodPolicy = commonv1.CleanPodPolicyNone
 )
 
-// NewReconciler creates a PyTorchJob Reconciler
+// Define all the prometheus counters for mxjobs
+var (
+	mxJobsCreatedCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "training_operator_mxjobs_created_total",
```

Comment:
do we want `mxjobs` part of the metrics name or a label?

Reply:
I have the same comment.  which would be the prometheus query here in both ways?